style.css

```diff
@@ -780,6 +765,47 @@ textarea::placeholder {
   margin-bottom: 8px;
 }
 
+.menu-open {
+  display: block;
+  height: 100vh;
+  width: 100%;
+  position: fixed;
+  background-color: #6070FF;
+  top: 0;
+  left: 0;
+  padding: 40px;
+}
+
```

Comment:
- [ ] `optional`
- [ ] Good so far :clap: :clap: 
- [ ] I suggest you make the background blur as the figma template, why? because pay attention to details will make you a better programmer. please see screenshot below:

`Your image`
![Screenshot from 2022-07-20 12-21-14](https://user-images.githubusercontent.com/66230306/179960886-a46a5cde-8147-4c61-b19f-9a3e4d435c6a.png)

`figma image`
![Screenshot from 2022-07-20 12-21-47](https://user-images.githubusercontent.com/66230306/179961116-0c95b6b2-83a1-4f3c-8483-e4c3e2c8119b.png)